wire-compiler/src/main/java/com/squareup/wire/schema/Target.kt

```diff
@@ -115,8 +115,12 @@ sealed class Target {
     /** True for emitted types to implement APIs for easier migration from the Java target. */
     val javaInterop: Boolean = false,
 
+    /** True for emitted services to provide blocking APIs only. */
+    // TODO(benoit) use it
+    val blockingServices: Boolean = false,
```

Comment:
can we move the introduction of this parameter to the PR that provides its implementation?